rwengine/include/engine/GameInputState.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef RWENGINE_GAMEINPUTSTATE_HPP
+#define RWENGINE_GAMEINPUTSTATE_HPP
+
+struct GameInputState
+{
+	/// Inputs that can be controlled
+	/// @todo find any sensible values
+	enum Control {
+		/* On Foot */
+		FireWeapon = 0,
+		NextWeapon,
+		NextTarget = NextWeapon,
+		LastWeapon,
+		LastTarget = LastWeapon,
+		GoForward,
+		GoBackwards,
+		GoLeft,
+		GoRight,
+		ZoomIn,
+		ZoomOut,
+		EnterExitVehicle,
+		ChangeCamera,
+		Jump,
+		Sprint,
+		AimWeapon,
+		Crouch,
+		Walk,
+		LookBehind,
+
+		/* In Vehicle */
+		VehicleFireWeapon = FireWeapon,
+		VehicleAccelerate,
```

Comment:
Sorry to bug about these again...
I think you missed my point last time (or I missed yours?).

I was asking about aliases like "VehicleFireWeapon = FireWeapon," for *all* enum members.
At the moment VehicleBrake would be a different index than ZoomIn and Jump. However, in the script these [all map to the very same button](http://www.gtamodding.com/wiki/00E1) (because R* used the PS2 layout as the default).

We'll have to decide wether we want an enum for each game function OR if we want to do it like R* (array of 20 inputs).
I proposed the change, thinking that currentLevels would map to the table linked above.